cdap-data-fabric/src/test/java/co/cask/cdap/spi/data/table/field/FieldTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data.table.field;
+
+import co.cask.cdap.api.common.Bytes;
+import com.google.common.collect.ImmutableSet;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Set;
+
+/**
+ *
+ */
+public class FieldTest {
+  @Test
+  public void testHashcode() {
+    Assert.assertEquals(generateFieldsSet(), generateFieldsSet());
+  }
+
+  @Test
+  public void testToString() {
+    System.out.println(generateFieldsSet());
```

Comment:
was this left here by accident?

Reply:
No, this test is to make sure that `toString()` does not throw any exception on bytes conversion to string.

Reply:
Wouldn't the proper way then be to do in a try catch block and assert if exception is caught.

Reply:
The test will fail if an exception is thrown. No need to catch it.

Reply:
OK if you want to test that toString() actually prints the contents of a byte array rather than the default (address in heap), why not create to equal objects, and assert that toString() on both of them yields the same string?

I think in general, println() is not a good practice. If you need to produce output, you can use logging.

Reply:
Made the suggested change